HW04/ekaterina_balabanovich/lib/task_solver.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module TaskSolver
+  attr_reader :homeworks
+
+  def get_homework(homework)
+    @homeworks << homework
+  end
+
+  def delete_homework(homework)
+    @homeworks.delete(homework)
+  end
```

Comment:
If it's possible would be better not to use instance variables
```suggestion
  def get_homework(homework)
    homeworks << homework
  end

  def delete_homework(homework)
    homeworks.delete(homework)
  end
```